misc/dnf-qubes-hooks.py

```diff
@@ -35,7 +35,10 @@ def __init__(self, base, cli):
         self.log = logging.getLogger('dnf')
 
     def transaction(self):
-        config = self.read_config(self.base.conf, PLUGIN_CONF)
+        try:
+            config = self.read_config(self.base.conf, PLUGIN_CONF)
+        except:
```

Comment:
This will catch any exception, not only related to wrong parameters. Ideally, there should be some condition checking dnf version. But if not possible, at least add `TypeError` here.